### PR TITLE
feat: add feedback log api

### LINKED ISSUE #1459: Post all the user parameters on giving the rating in the chat
**Feature**

The analytics team is implementing an API to add the feedback logs that will begin the usage of ML in SUSI, so on each rating we have to make a post request to the server with all the user parameters.

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by batbrain7**
Fixes #1459

### CODE REVIEW

**Comment by codedsun** [comment]
@batbrain7 Check the build failing.

File: app/src/main/java/org/fossasia/susi/ai/rest/services/SusiService.java [hunk 1]
```diff
@@ -200,4 +199,23 @@
...
+     *              feedback       the feedback thumbsup or thumbsdown
+     *              user_query     the query asked by user
+     *              susi_reply     the message replied by susi
+     *              country_name   the country name
+     *              country_code   the country code
+     *              device_type    the type of the device
+     * @return the susi response
+     */
+    @GET("/cms/feedbackLog.json")
+    Call<FeedBackLogData> postFeedBackData(@QueryMap Map<String,String> query);
```
[the remark this answers is not in the document]
Comment (batbrain7) [reply app/src/main/java/org/fossasia/susi/ai/rest/services/SusiService.java]: @iamareebjamal this is done in clientBuilder

**Comment by batbrain7** [comment]
@codedsun check the label

**Comment by batbrain7** [comment]
@AnupKumarPanwar can you list what all is needed to sent corresponding to the parameters

File: app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/FeedBackLogData.kt [hunk 2]
```diff
@@ -0,0 +1,15 @@
+package org.fossasia.susi.ai.rest.responses.susi
+
+
```
Comment (arundhati24) [inline app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/FeedBackLogData.kt]: extra line

**Comment by arundhati24** [comment]
@batbrain7 What's the status of this PR?

**Pull request closed without merging**